changelogs/118-bugfix-info_datacenters-in-documentation

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
I think this file should be added to "changelogs/fragments/", not to "changelogs" itself.

Reply:
also missing the .yml extension